VideoLocker/src/main/java/org/edx/mobile/model/IVertical.java

```diff
@@ -0,0 +1,13 @@
+package org.edx.mobile.model;
+
+
+import java.util.List;
+
+public interface IVertical extends IComponent{
```

Comment:
@aleffert a good thing to use interface is to support multiple inheritance. if after the course outline api is finalized and we find interface approach is overkilling, i will remove it